### ISSUE TITLE
External trigger example gives error when trigger pin is pulled high

### ISSUE THREAD

**Issue by ehong-tl** [ArduCAM/ArduCAM_USB_Camera_Shield#67]
Hi,

I'm using MT9N001 camera with USB2 shield board.

I've trouble capturing image using external trigger image capture example in new SDK.

I keep getting alternating error as follows each time when I pulled the trigger pin high.

_Take picture.
('Take picture fail, ret_val = ', 65318)
Take picture.
('Take picture fail, ret_val = ', 65316)
Take picture.
('Take picture fail, ret_val = ', 65318)
Take picture.
('Take picture fail, ret_val = ', 65316)
Take picture.
('Take picture fail, ret_val = ', 65318)
Take picture.
('Take picture fail, ret_val = ', 65316)_

May I know what's going on here, and how to fix it?

**Comment by ArduCAM** [comment]
Would you please let me know your hardware setup?  How do you send the trigger signals? Please send to support@arducam.com

**Comment by ehong-tl** [comment]
Hi,

I didn't change much in terms of hardware setup.
I send the trigger signal by just connecting Vcc pin to Trigger pin with jumper wire to pull it high.
![img_20181010_134753](https://user-images.githubusercontent.com/28379826/46715872-636b4280-cc94-11e8-96c2-33f92c656a21.jpg)

**Comment by ArduCAM** [comment]
It is saft to trigger this signal with pulse from Arduino like GPIO, the trigger input default should be logic low, and pull logic high for 10uS or so to trigger the capture.

**Comment by ehong-tl** [comment]
Hi, I've tried to send 10us pulse using Arduino to the camera, but there is no response, it 'doesn't even print out _Take picture._ line.

**Comment by ArduCAM** [comment]
I realized that your camera only has 8MByte frame buffer, it can't hold 9MP image sensor. We will build 32MByte frame buffer version later.

**Comment by ehong-tl** [comment]
So the current USB2 shield with 8 MB buffer does not support external trigger image capture?

**Comment by ArduCAM** [comment]
No, I think it supports external trigger, but the MT9N001 9MP camera has exceeded the maximum frame buffer of current camera shield.

**Comment by ehong-tl** [comment]
Hi @habib-Boloorchi ,

I connect one of Arduino's digital pin to Trigger pin on the camera board (not the USB shield) and Arduino's ground pin to camera board's ground pin.
